whipper/common/mbngs.py

```diff
@@ -250,11 +325,19 @@ def _getMetadata(release, discid=None, country=None):
                     track.artist = trackCredit.getName()
                     track.sortName = trackCredit.getSortName()
                     track.mbidArtist = trackCredit.getIds()
+                    track.recordingArtist = recordingCredit.getName()
 
                     track.title = t.get('title', t['recording']['title'])
                     track.mbid = t['id']
                     track.mbidRecording = t['recording']['id']
-                    track.mbidWorks = _getWorks(t['recording'])
+                    works = _getWorks(t['recording'])
```

Comment:
This line looks weird here. I’d place this line outside of the `track.… = …` lines. Probably up in the same "chunk" where `recordingCredit` and `trackCredit` get defined.

Reply:
This should be resolved by now.